demos-go/cb-mpc-go/api/mpc/secure.go

```diff
@@ -0,0 +1,50 @@
+package mpc
+
+import (
+	"crypto/hmac"
+	"crypto/sha256"
+	"crypto/subtle"
+)
+
+// ExpandFromSeed deterministically expands a 32-byte seed into l bytes.
+// If l <= 32, it returns the first l bytes. If l > 32, it uses HMAC-SHA256
+// with a counter to derive additional pseudorandom blocks securely.
+// This is suitable for seeding or deriving per-encapsulation randomness.
+func ExpandFromSeed(seed [32]byte, l int) []byte {
```

Comment:
It seems like we could have used the functions in this file (expanding from seed and secure wipe) in the rsaDemoKem of the pve demo code, right?